### PR TITLE
docs: Add L2 ETH to important_addresses.adoc.

### PR DESCRIPTION
**Pull request by stoobie**
### Description of the Changes

This PR addresses https://starkware.monday.com/boards/2575505837/pulses/5185579645/posts/2420014350, *Fee token address*

### PR Preview URL

After you push a commit to this PR, a preview is built and a URL to the root of the preview appears in the comment feed.

Paste here the specific URL(s) of the content that this PR addresses.

### Check List

- [x] Changes have been done against dev branch, and PR does not conflict
- [x] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/930)

### CODE REVIEW

**Comment by JameStark** [comment]
LGTM @stoobie

**Review by JameStark — APPROVED** [review]

**Comment by yair-starkware** [comment]
*[`components/Starknet/modules/tools/pages/important_addresses.adoc` line 18 at r3](https://reviewable.io/reviews/starknet-io/starknet-docs/930#-NjGsrGb57ZKEQx8AVtT:-NjGsrGc0VNw-p85ODu9:b-ee7utv) ([raw file](https://github.com/starknet-io/starknet-docs/blob/a21306409eeae25fdac43ba032527542f9788ccf/components/Starknet/modules/tools/pages/important_addresses.adoc#L18)):*
> ```AsciiDoc
> Sequencer base URL for API routing:: \https://alpha4.starknet.io
> 
> == Starknet fee token (Currently ETH)
> ```

I suggest mentioning that it is the same address in all environments



_Code quote:_
```AsciiDoc
Starknet fee token (Currently ETH)
```

**Review by stoobie — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/930)* status: 0 of 1 files reviewed, 3 unresolved discussions (waiting on @yair-starkware)

___
*[`components/Starknet/modules/tools/pages/important_addresses.adoc` line 18 at r3](https://reviewable.io/reviews/starknet-io/starknet-docs/930#-NjGsrGb57ZKEQx8AVtT:-NjGwpC88m8OD7LRoKBu:b-mq8kml) ([raw file](https://github.com/starknet-io/starknet-docs/blob/a21306409eeae25fdac43ba032527542f9788ccf/components/Starknet/modules/tools/pages/important_addresses.adoc#L18)):*
>all environments

Do you Mean both testnet (Sepolia, Goerli, etc) and Mainnet?

**Review by stoobie — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/930)* status: 0 of 1 files reviewed, all discussions resolved (waiting on @yair-starkware)

___
*[`components/Starknet/modules/tools/pages/important_addresses.adoc` line 18 at r3](https://reviewable.io/reviews/starknet-io/starknet-docs/930#-NjGsrGb57ZKEQx8AVtT:-NjIWYF44NR5btgoeTdp:b-896fix) ([raw file](https://github.com/starknet-io/starknet-docs/blob/a21306409eeae25fdac43ba032527542f9788ccf/components/Starknet/modules/tools/pages/important_addresses.adoc#L18)):*
<details><summary><i>Previously, stoobie (Steve Goodman) wrote…</i></summary><blockquote>

>all environments

Do you Mean both testnet (Sepolia, Goerli, etc) and Mainnet?
</blockquote></details>

Done.

**Review by yair-starkware — COMMENTED** [review]
:lgtm:

> Reviewed 1 of 1 files at r3, 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/930#-:-NjIo4FqCaU1Pg9pHi20:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @stoobie)

File: components/Starknet/modules/tools/pages/important_addresses.adoc [hunk 1]
```diff
@@ -9,12 +9,18 @@ The Starknet core contract:: link:https://etherscan.io/address/0xc662c410C0ECf74
...
 Sequencer base URL for API routing:: \https://alpha-mainnet.starknet.io
 
-== Starknet Alpha version on Goerli testnet 1
+== Starknet Alpha version on Goerli testnet
 
 The Starknet core contract:: link:https://goerli.etherscan.io/address/0xde29d060D45901Fb19ED6C6e959EB22d8626708e[`0xde29d060D45901Fb19ED6C6e959EB22d8626708e`^]
 Verifier address::  link:https://goerli.etherscan.io/address/0x8f97970aC5a9aa8D130d35146F5b59c4aef57963[`0x8f97970aC5a9aa8D130d35146F5b59c4aef57963`^]
 Sequencer base URL for API routing:: \https://alpha4.starknet.io
 
+== Starknet fee token (Currently ETH)
```
Comment (stoobie) [inline components/Starknet/modules/tools/pages/important_addresses.adoc:18]:
```suggestion
== Starknet fee token

The Starknet fee token is currently ETH.
```

**Pull request merged**